tests/test-data/projects/no-docker-compose/dockerignore2/symlink-b.txt

```diff
@@ -0,0 +1 @@
+src/src-b.txt
```

Comment:
`symlink-a.txt` and `symlink-b.txt` are symbolic links rather than regular files. I found that they checkout as regular files on Windows with MSYS2, but it's OK, the tests are happy with this all the same. On Linux and macOS, the symbolic links are tested as symbolic links.